README.md

```diff
@@ -1,89 +1,134 @@
+1. Clone this Github repository:
 
-An example configuration file can be found in [`config/example-connector.properties`](config/example-connector.properties).
+    `git clone git@github.com:ably/kafka-connect-ably.git`
 
-# Running locally
+2. Build the connector using [Maven](https://maven.apache.org/):
 
-The easiest way to run the connector locally is using [Docker Compose](https://docs.docker.com/compose/).
+    `mvn clean package`
 
-This repository includes a [docker-compose.yml](docker-compose.yml) file which uses the
-[Confluent Platform Docker images](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
+3. A `.zip` file will be produced in the `/target/components/packages/` folder after the process has run.
 
-To start the Docker Compose cluster, first copy the example properties file to
-`config/docker-compose-connector.properties` and set `client.key` to your Ably
-API key:
+4. Install the `.zip` into a directory specified in the `plugin.path` of your connect worker's configuration properties file. See the [Confluent instructions](https://docs.confluent.io/home/connect/install.html#install-connector-manually) for further information on this step.
 
-```
-cp config/example-connector.properties config/docker-compose-connector.properties
-```
+5. [Configure](#configuration) the connector.
 
-Then start the cluster by running:
+### Docker
 
-```
-docker-compose up -d
-```
+There is a [`docker-compose.yml`](docker-compose.yml) file included in this repository that can be used to run the connector locally using [Docker Compose](https://docs.docker.com/compose/). The Docker Compose file is based on the [Confluent Platform Docker images](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
 
-You can check the logs of the connector with:
+1. Create and configure a [configuration file](#configuration) ensuring you have set at least the basic properties.
 
-```
-docker-compose logs connector
-```
+2. Start the cluster using:
 
-To check the connector is working, subscribe to messages on the `kafka-connect-ably-example`
-Ably channel in a separate terminal window (this uses [SSE](https://ably.com/documentation/sse)
-for convenience, but you could also use an [Ably SDK](https://ably.com/download)):
+    `docker-compose up -d`
 
-```
-curl -s -u "<your Ably API key>" "https://realtime.ably.io/sse?channel=kafka-connect-ably-example&v=1.1"
-```
+    **Note**: You can view the logs using `docker-compose logs connector`
 
-Produce some messages to the `kafka-connect-ably-example` Kafka topic using the
-`kafka-console-producer` CLI tool in the Kafka container:
+3. Once the pods have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
```

Comment:
This should start "Once the containers have started" ("pods" is a Kubernetes specific term, and there is no Kubernetes involved here)

Reply:
Good point - thank you!